vllm/engine/arg_utils.py

```diff
@@ -709,7 +741,8 @@ def create_engine_config(self, ) -> EngineConfig:
             "CPU offload space must be non-negative"
             f", but got {self.cpu_offload_gb}")
 
-        multimodal_config = MultiModalConfig()
+        multimodal_config = MultiModalConfig(
+            limit_per_prompt=self.limit_mm_per_prompt or {})
```

Comment:
Doesn't this mean for every single-image model, now user will need to pass  `--limit-mm-per-prompt {"image": 1}`?

Reply:
No, the current default for all modalities is set to `1`.

Reply:
I have updated the help text, see if it is clearer to you now.